src/vs/editor/contrib/wordOperations/browser/wordOperations.ts

```diff
@@ -183,7 +183,12 @@ export class CursorWordAccessibilityLeft extends WordLeftCommand {
 			inSelectionMode: false,
 			wordNavigationType: WordNavigationType.WordAccessibility,
 			id: 'cursorWordAccessibilityLeft',
-			precondition: undefined
+			precondition: undefined,
+			kbOpts: {
+				kbExpr: ContextKeyExpr.and(EditorContextKeys.textInputFocus, CONTEXT_ACCESSIBILITY_MODE_ENABLED, IsWindowsContext, ContextKeyExpr.equals('focusedView', 'workbench.panel.output')),
```

Comment:
No const we can use for either of these strings?

Reply:
import violation

Reply:
In that case these commands should live where this would not give an import violation, otherwise it's just hiding the soft layer breaker where you're depending upon the panel (workbench) from the editor (editor/contrib/). This does have real consequences in that this keybinding would be included into the bundled monaco editor which will never know about that panel.